tests/functional/adapter/test_basic.py

```diff
@@ -51,4 +53,13 @@ class TestSnapshotCheckColsTrino(BaseSnapshotCheckCols):
 
 @pytest.mark.xfail(reason="Snapshot not supported in dbt-trino")
 class TestSnapshotTimestampTrino(BaseSnapshotTimestamp):
+    pass   
+
+
+class TestTrinoValidateConnection(BaseValidateConnection):	
+    pass 	    
+
+
+@pytest.mark.xfail(reason="Docs generation not supported in dbt-trino")
```

Comment:
Is it related to https://github.com/starburstdata/dbt-trino/pull/77 ?

Reply:
Yes it would fail as we don't yet support docs generation